examples/templates/README.md

```diff
@@ -1,21 +1,45 @@
 ## Templates Guidline
 
-These templates are a base for the new code examples.
+This folder contains templates for new code examples. Each new code example should be created using the template from this folder. We have templates for examples made in React, Angular, Vue, and of course Vanilla JavaScript.
+
+#### Prerequisite
+To work with code examples (and Handsontable development in general) use at least Node version `15.9.0`. If you have Node Version Manager installed, you can just type in the terminal `nvm use`.
 
 ### Creating a new example
 
-All new code examples should be added to the `/examples/next` directory. Based on them you can create versioned code examples.
+In this guideline, we will walk you through the process of creating new code examples. 
+
+> If you need some inspiration, what can be done with code examples, go ahead see existing examples in `examples/next/docs/` to get inspired!
+
+First off, create a new branch in order to keep the development process clean.
+
+Inside the `examples/templates` directory where all templates for future code examples live. You can see that we have templates for examples made in React, Angular, Vue, and of course vanilla JavaScript.
+
+To create a new code example in Vanilla JavaScript to be used in the Handsontable official documentation, we need to make use of the template from the `examples/templates/js` directory.
+
+#### 1. Use the existing template
```

Comment:
Clone/Duplicate/Copy the existing template?

I'm not sure here. I would use `copy`.